.gitlab/common/shared.yml

```diff
@@ -56,3 +56,8 @@
       export GITHUB_INSTALLATION_ID=$($CI_PROJECT_DIR/tools/ci/aws_ssm_get_wrapper.sh $MACOS_GITHUB_INSTALLATION_ID_2_SSM_NAME)
       echo "Using GitHub App instance 2"
     fi
+
+# Install `dd-pkg` and lint packages produced by Omnibus, supports only deb and rpm packages
+.lint_linux_packages:
+  - curl -sSL "https://dd-package-tools.s3.amazonaws.com/dd-pkg/${DD_PKG_VERSION}/dd-pkg_Linux_${DD_PKG_ARCH}.tar.gz" | tar -xz -C /usr/local/bin dd-pkg
```

Comment:
Could we use `$(uname -m)` instead of `DD_PKG_ARCH`?
For x86_64 I'm sure that would work, but not entirely for `arm64`

Reply:
I could change how `dd-pkg` is published, but that would also required updating everywhere it's been setup so far (which isn't awful). But I'd rather get this out, and then iterate some.

Reply:
Changing the way we release our tooling to save an environment variable seems overkill. I guess you could have a small if/else to attempt at dispatching the correct arch based on the OS, but I'm not sure this will make things clearer